manifests/database.pp

```diff
@@ -16,6 +16,8 @@
     refreshonly => false,
   }
 
+  pulpcore::admin { 'reset-admin-password --password' }
```

Comment:
This doesn't look very idempotent. There's also no guarantee this actually runs after the migration now.